scripts/delta-parallel-spawner.sh

```diff
@@ -2,8 +2,8 @@
 #
 # Used to wrap the spawning of parallel mirgecom drivers on Delta.
 # unset CUDA_CACHE_DISABLE
-POCL_CACHE_ROOT=${POCL_CACHE_ROOT:-"/projects/bbkf/$USER/pocl-scratch"}
-XDG_CACHE_ROOT=${XDG_CACHE_HOME:-"/projects/bbkf/$USER/xdg-scratch"}
+POCL_CACHE_ROOT=${POCL_CACHE_ROOT:-"/scratch/bbkf/$USER/pocl-scratch"}
```

Comment:
This is hard-coding to a specific project, which is not ideal.

Reply:
Right, is there a way to 'get' the project, either via env variable or another command?

Reply:
> Right, is there a way to 'get' the project, either via env variable or another command?

I think something like this:
```
(omega.main) mtcampbe@Mikes-MacBook-Pro-2 examples % cat ../scripts/lassen-parallel-spawner.sh 
#!/bin/bash
#
# Used to wrap the spawning of parallel mirgecom drivers on Lassen
# unset CUDA_CACHE_DISABLE
POCL_CACHE_ROOT=${POCL_CACHE_ROOT:-"/tmp/$USER/pocl-scratch"}
XDG_CACHE_ROOT=${XDG_CACHE_HOME:-"/tmp/$USER/xdg-scratch"}
POCL_CACHE_DIR=${POCL_CACHE_DIR:-"${POCL_CACHE_ROOT}/rank$OMPI_COMM_WORLD_RANK"}
XDG_CACHE_HOME=${XDG_CACHE_HOME:-"${XDG_CACHE_ROOT}/rank$OMPI_COMM_WORLD_RANK"}
export POCL_CACHE_DIR
export XDG_CACHE_HOME

"$@"
```
is better.  It automatically sets it to `/tmp` which is available on all platforms, and also allows the user to override by setting `POCL_CACHE_ROOT` and `XDG_CACHE_ROOT`.